### PR TITLE
Fixes network issues on AIX

### PR DESCRIPTION
**Pull request by alorbach**
- Add handling for other ERRNO codes in tcp.c for AIX
- Fix commands in some tests that were not correctly handled on AIX.

### CODE REVIEW

File: src/tcp.c [hunk 1]
```diff
@@ -3382,7 +3383,11 @@ relpTcpSend(relpTcp_t *const pThis, relpOctet_t *const pBuf, ssize_t *const pLen
 		if(written == -1) {
 			switch(errno_save) {
 				case EAGAIN:
+#if defined(_AIX)
+				// AIX Workarround handling other ERRNO codes
+				case 0:
 				case EINTR:
```
Comment (kortemik) [inline src/tcp.c:3389]: This encapsulates case EINTR: within defined(_AIX), is this intended?

File: src/tcp.c [hunk 2]
```diff
@@ -3217,8 +3217,9 @@ relpTcpRcv(relpTcp_t *const pThis, relpOctet_t *const pRcvBuf, ssize_t *const pL
 			} else if(errno == ECONNRESET) {
 				pThis->pEngine->dbgprint((char*)"relpTcpRcv: read failed with errno ECONNRESET!\n");
 #if defined(_AIX)
-			} else if(errno == 0) {
-				// Set mode to Retry if errno is 0, this actually happens on AIX 10.x
+			} else if(	errno == 0 ||
+						errno == EINTR) {
```
Comment (kortemik) [inline src/tcp.c:3221]: errno == EINPROGRESS is missing. I think I did see that from relpTcpRcv even though man page differs on codes of recv

File: src/tcp.c (hunk 1 above)
Comment (alorbach) [reply src/tcp.c:3389]: > This encapsulates case EINTR: within defined(_AIX), is this intended?

Yes as far as I know we do not have "EINTR" in errno on other systems, typically it is set to "EAGAIN".

File: src/tcp.c (hunk 2 above)
Comment (alorbach) [reply src/tcp.c:3221]: Did you receive that code during testing? According to DOC, this code only being used in connect() and not in recv() / send().

Comment (kortemik) [reply src/tcp.c:3221]: I think assignment of 
errno = EINPROGRESS;
caused this as a side effect.

Comment (alorbach) [reply src/tcp.c:3221]: > I think assignment of errno = EINPROGRESS; caused this as a side effect.

perhaps yes, but that assignment was removed so I think we should be good now with this PR.

Comment (kortemik) [reply src/tcp.c:3221]: i agree. i didn't observe EINPROGRESS in recv / send after removal

**Review by kortemik — APPROVED** [review]

**Pull request merged**